android/src/main/java/com/klaviyoreactnativesdk/KlaviyoReactNativeSdkModule.kt

```diff
@@ -60,16 +61,28 @@ class KlaviyoReactNativeSdkModule(
   }
 
   @ReactMethod
-  fun registerForInAppForms() {
+  fun registerForInAppForms(configuration: ReadableMap?) {
```

Comment:
should the type be a mirror of the native sdk type?

Reply:
something like - https://github.com/klaviyo/klaviyo-react-native-sdk/blob/master/src/Event.ts#L77

Reply:
also, I see that `setProfile` doesn't take in a custom type and does `ReadableMap`. So I could be making an invalid suggestion.

Reply:
nvm this is the native module and not the api. I will show myself out! 

![](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExNzd3MXBrMmE2YXhzMGhid3hyb3hsazF2OTR4dnV0bXBvYXo2MHU4YyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/l0K42RIaNOZcK7CNy/giphy.gif)

Reply:
Here I was about to comment exactly the same thing